js/src/features/project/types.d.ts

```diff
@@ -89,15 +89,40 @@ declare module "ProjectModels" {
   export interface ProjectTeam {
     id: string;
     name: string;
-    status: ProjectTeamStatus;
+    status: ProjectTeamStatusResponse;
     members?: ProjectTeamMember[];
```

Comment:
Almost never makes sense to makes array optional. Empty array is the best initial value.

Reply:
fix